Library/Formula/emacs.rb

```diff
@@ -26,6 +26,7 @@ class Emacs < Formula
 
   option "with-cocoa", "Build a Cocoa version of emacs"
   option "with-ctags", "Don't remove the ctags executable that emacs provides"
+  option "with-libxml2", "Build with libxml2 support"
```

Comment:
Thanks for your hint, I tried to do that but when I do a `brew info emacs` it doesn't show the changed option. when I do the same thing with another package (`lua51`) I can change the option quite easily - probably doing something silly that I can't see at the moment, but I'll get this done. :smile: